services/livesync-service-base.ts

```diff
@@ -184,7 +184,7 @@ class LiveSyncServiceBase implements ILiveSyncServiceBase {
 					}
 
 					if (device.applicationManager.canStartApplication() && !shouldRefreshApplication) {
-						await device.applicationManager.startApplication(appIdentifier);
+						await device.applicationManager.startApplication({ appId: appIdentifier, projectName: "" });
```

Comment:
Can `projectName` be an optional param? This way we can avoid passing an empty param here

Reply:
Same as above - this service is not used anymore. We'll delete it soon.